contract_api/serverless.yml

```diff
@@ -9,7 +9,7 @@ provider:
   description: Description of what the lambda function does # optional, Description to publish to AWS
   memorySize: 128
   timeout: 30
-  region: us-east-1
+  region: ${file(./config.${self:provider.stage}.json):REGION}
```

Comment:
@vinthedark can we add this config for mainnet configuration. we will deploy this changes tomorrow